suite/runsuite.cmake

```diff
@@ -546,6 +552,33 @@ if (ARCH_IS_X86 AND UNIX AND (a64_on_x86_only OR NOT arg_automated_ci))
     " OFF ${arg_package} "")
 endif ()
 
+if (ARCH_IS_X86 AND UNIX)
+  # TODO i#3544: Run tests under QEMU
+  set(orig_extra_ctest_args ${extra_ctest_args})
+  set(run_tests OFF)
```

Comment:
This seems like a bug: we're saving prev_run_tests (at line 560 below) *after* modifying it here. Perhaps this should be moved to below L560?